### PR TITLE
Added a What Remains of Edith Finch guide and corrected something in the DmC guide

### PR DESCRIPTION
**Pull request by originalnicodr**

### CODE REVIEW

File: markdown/GameGuides/dmc.md [hunk 1]
```diff
@@ -1,4 +1,7 @@
-![DmC: Devil May Cry](Images\dmc.png "Shot by White Harpy")
+![DmC: Devil May Cry](\\dmc.png "Shot by White Harpy")
```
Comment (FransBouma) [inline markdown/GameGuides/dmc.md]: This stuff doesn't work, just use format 'Images\imagename.png/jpg "title"'

File: markdown/GameGuides/dmc.md [hunk 2]
```diff
@@ -110,7 +113,7 @@ Keep in mind that you can use the combination of keys that fits you best.
 
 In addition you can also change the default anisotropic filtering (4) up to 16 & Default smoothing (HD AA) in the config file `devilengine.ini` or use the antialising from the nvidia panel.
 
-Hotsampling
+Hotslamping
```
Comment (FransBouma) [inline markdown/GameGuides/dmc.md]: It really is Hotsampling, not Hotslamping, sorry ;)

**Comment by originalnicodr** [comment]
My bad, the image reference was fine in the previous markdown file but the one I edited was another one.

**Comment by FransBouma** [comment]
No worries, I fixed it. 

Funny you corrected 'Hotsampling' back to 'Hotslamping' :D

**Pull request merged**